Update commit 35044e8

offhub opened the pull request:
#126 #133

- moved files to Content:
sandboxielogon.md (renamed as SandboxieLogon.md)
feature-comparison.md (renamed as FeatureComparison.md)

- moved files to PlusContent:
RamDiskSandboxes.md (renamed as RamDiskSupport.md)
EncryptedSandboxes.md (renamed as BoxEncryption.md)

- deleted files:
changelog.md (unnecessary)
contribute.md (use CONTRIBUTING.md)
privacy-policy.md (website only)

- further additions/improvements/fixes

offhub committed:
update FeatureComparison.md

isaak654 commented on Content/FeatureComparison.md:
```diff
@@ -1,28 +1,34 @@
+|---------------------------------------------------------------------|-----------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------|------------------|----------------------------------------------------------------------------------------------|------------------------ ...
+| Usage                                                               |                                                                                         |                                                                                                   |                  | Personal                                                                                     | Personal                ...
+| Support reminder                                                    | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) Yes | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) No          |                  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) Yes      | ![Green Color](https:// ...
+| PC's per Certificate                                                |                                                                                         | As Certified                                                                                      |                  |                                                                                              | Personal\*\*            ...
```
```suggestion
| PCs per Certificate                                                |                                                                                         | As Certified                                                                                      |                  |                                                                                              | Personal\*\*                                                                                             | Personal                                                                                                         | Personal and Family\*\*\*                                                                                | 1                                                                                                        | Personal and Family                                                                       |
```

isaak654 commented on Content/FeatureComparison.md:
```diff
@@ -1,28 +1,34 @@
+| Windows 11 Context menu                                             | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No            |                  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes    | ![Green Color](https:// ...
+| [Box Snapshots](../PlusContent/BoxSnapshots.md)                     | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No            |                  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes    | ![Green Color](https:// ...
+| [WFP support](../PlusContent/WFPSupport.md)                         | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes (no UI) |                  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes    | ![Green Color](https:// ...
+| [Privacy enchanced boxes](../PlusContent/privacy-mode.md)           | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes (no UI) |                  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No       | ![Green Color](https:// ...
```
```suggestion
| [Privacy enhanced boxes](../PlusContent/privacy-mode.md)           | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes (no UI) |                  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No       | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                        | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes |
```

isaak654 commented on Content/FeatureComparison.md:
```diff
@@ -1,28 +1,34 @@
+| [Box Snapshots](../PlusContent/BoxSnapshots.md)                     | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No            |                  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes    | ![Green Color](https:// ...
+| [WFP support](../PlusContent/WFPSupport.md)                         | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes (no UI) |                  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes    | ![Green Color](https:// ...
+| [Privacy enchanced boxes](../PlusContent/privacy-mode.md)           | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes (no UI) |                  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No       | ![Green Color](https:// ...
+| [Security Enchanced boxes](../PlusContent/security-mode.md)         | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes (no UI) |                  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No       | ![Green Color](https:// ...
```
```suggestion
| [Security enhanced boxes](../PlusContent/security-mode.md)         | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes (no UI) |                  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No       | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                        | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes |
```

isaak654 commented on Content/FeatureComparison.md:
```diff
@@ -1,28 +1,34 @@
+| [WFP support](../PlusContent/WFPSupport.md)                         | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes (no UI) |                  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes    | ![Green Color](https:// ...
+| [Privacy enchanced boxes](../PlusContent/privacy-mode.md)           | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes (no UI) |                  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No       | ![Green Color](https:// ...
+| [Security Enchanced boxes](../PlusContent/security-mode.md)         | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes (no UI) |                  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No       | ![Green Color](https:// ...
+| [Compatibility enchanced boxes](../PlusContent/compartment-mode.md) | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes (no UI) |                  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No       | ![Green Color](https:// ...
```
```suggestion
| [Compatibility enhanced boxes](../PlusContent/compartment-mode.md) | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes (no UI) |                  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No       | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                        | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes |
```

isaak654 commented on Content/FeatureComparison.md:
```diff
@@ -1,28 +1,34 @@
+| [Ram Disk Integration](../PlusContent/RamDiskSandboxes.md)          | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes (no UI) |                  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No       | ![Green Color](https:// ...
+
+
+\*) A personal type certificate once expired does not unlock features in builds compiled after it expired, builds compiled before that time retain their exclusive features unlocked state.  
```
```suggestion
\*) A personal type certificate once expired does not unlock features in builds compiled after it expired, builds compiled before that time retain their exclusive features in unlocked state.
```

isaak654 commented on Content/FeatureComparison.md:
```diff
@@ -1,28 +1,34 @@
+
+
+\*) A personal type certificate once expired does not unlock features in builds compiled after it expired, builds compiled before that time retain their exclusive features unlocked state.  
+\*\*) Personal covers all devices you use yourself, so if you have a laptop and a desktop one certificate covers booth.  
```
```suggestion
\*\*) Personal covers all devices you use yourself, so if you have a laptop and a desktop, one certificate covers both.
```

isaak654 commented on Content/FeatureComparison.md:
```diff
@@ -1,28 +1,34 @@
+
+\*) A personal type certificate once expired does not unlock features in builds compiled after it expired, builds compiled before that time retain their exclusive features unlocked state.  
+\*\*) Personal covers all devices you use yourself, so if you have a laptop and a desktop one certificate covers booth.  
+\*\*\*) A family pack can be used for the entire family it is not required for the family to live in teh same household, so the cert can be used for children which moved out or your grand parents.  
```
```suggestion
\*\*\*) A family pack can be used for the entire family, it is not required for the family to live in the same household, so the certificate can be used for children who have moved out or your grandparents.
```

isaak654 commented on Content/FeatureComparison.md:
```diff
@@ -1,28 +1,34 @@
+\*) A personal type certificate once expired does not unlock features in builds compiled after it expired, builds compiled before that time retain their exclusive features unlocked state.  
+\*\*) Personal covers all devices you use yourself, so if you have a laptop and a desktop one certificate covers booth.  
+\*\*\*) A family pack can be used for the entire family it is not required for the family to live in teh same household, so the cert can be used for children which moved out or your grand parents.  
+\*\*\*\*) For as long as the service exists.  
```
```suggestion
\*\*\*\*) For as long as the service exists.
```

isaak654 commented on Content/FeatureComparison.md:
```diff
@@ -1,28 +1,34 @@
+| [Sandboxie-Live](../PlusContent/Sandboxie-Live.md)                  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No            |                  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No       | ![Yellow Color](https:/ ...
+| UI Dark mode                                                        | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No            |                  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes    | ![Green Color](https:// ...
+| Start Menu Integration                                              | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No            |                  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes    | ![Green Color](https:// ...
+| Windows 11 Context menu                                             | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No            |                  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes    | ![Green Color](https:// ...
```
```suggestion
| Windows 11 context menu                                             | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No            |                  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes    | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                        | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes |
```

isaak654 commented on Content/FeatureComparison.md:
```diff
@@ -1,28 +1,34 @@
+| Old builds work after expiration                                    |                                                                                         | As Certified                                                                                      |                  |                                                                                              | ![Red Color](https://pl ...
+| [Sandboxie-Live](../PlusContent/Sandboxie-Live.md)                  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No            |                  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No       | ![Yellow Color](https:/ ...
+| UI Dark mode                                                        | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No            |                  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes    | ![Green Color](https:// ...
+| Start Menu Integration                                              | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No            |                  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes    | ![Green Color](https:// ...
```
```suggestion
| Start Menu integration                                              | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No            |                  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes    | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                        | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes |
```

isaak654 commented on Content/FeatureComparison.md:
```diff
@@ -1,28 +1,34 @@
+| [Privacy enchanced boxes](../PlusContent/privacy-mode.md)           | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes (no UI) |                  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No       | ![Green Color](https:// ...
+| [Security Enchanced boxes](../PlusContent/security-mode.md)         | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes (no UI) |                  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No       | ![Green Color](https:// ...
+| [Compatibility enchanced boxes](../PlusContent/compartment-mode.md) | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes (no UI) |                  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No       | ![Green Color](https:// ...
+| ARM64 Support                                                       | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No            |                  | ![Orange Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff9900&text=+) Trial | ![Green Color](https:// ...
```
```suggestion
| ARM64 support                                                       | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No            |                  | ![Orange Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff9900&text=+) Trial | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                        | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes |
```

isaak654 commented on Content/FeatureComparison.md:
```diff
@@ -1,28 +1,34 @@
+| [Security Enchanced boxes](../PlusContent/security-mode.md)         | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes (no UI) |                  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No       | ![Green Color](https:// ...
+| [Compatibility enchanced boxes](../PlusContent/compartment-mode.md) | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes (no UI) |                  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No       | ![Green Color](https:// ...
+| ARM64 Support                                                       | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No            |                  | ![Orange Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff9900&text=+) Trial | ![Green Color](https:// ...
+| [Ram Disk Integration](../PlusContent/RamDiskSandboxes.md)          | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes (no UI) |                  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No       | ![Green Color](https:// ...
```
```suggestion
| [RAM Disk integration](../PlusContent/RamDiskSandboxes.md)          | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes (no UI) |                  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No       | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                        | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes |
```

isaak654 commented on Content/FeatureComparisonOld.md:
```diff
@@ -0,0 +1,28 @@
+# Feature Comparison (to be updated)
```
```suggestion
# Feature Comparison (obsolete)
```

isaak654 commented on Content/FeatureComparisonOld.md:
```diff
@@ -0,0 +1,28 @@
+
+**Please note that a Business Certificate is required to use Sandboxie Plus in a commercial or educational setting!**
+
+> ### ⚠️ The following table is obsolete as of [Sandboxie Plus 1.11.0](https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.11.0) and should be updated with the new one in [https://sandboxie-plus.com/feature-comparison/](https://sandboxie-plus.com/feature-comparison/)
```
```suggestion
> ### ⚠️ The following table is obsolete as of version [1.11.0 / 5.66.0](https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.11.0) and should be updated with the new one in [https://sandboxie-plus.com/feature-comparison/](https://sandboxie-plus.com/feature-comparison/)
```

isaak654 commented on Content/FeatureComparison.md:
```diff
@@ -1,28 +1,34 @@
+
+Starting with version 1.11.0 there is also a [Advanced Encryption Pack](https://xanasoft.com/product/sandboxie-plus-advanced-upgrade/) available which needs to be obtained in addition to a supporter certificate in order to use sandbox encryption.
+
+**Please note that to use Sandboxie Plus in a commercial or educational setting a [Business Certificate](https://xanasoft.com/product/sandboxie-plus-business/) is required!**
```
```suggestion
**Please note that a [Business Certificate](https://xanasoft.com/product/sandboxie-plus-business/) is required to use Sandboxie Plus in a commercial or educational setting!**
```

isaak654 commented on Content/FeatureComparison.md:
```diff
@@ -1,28 +1,34 @@
+
+Some exclusive functionality is only available to project supporters with a valid [Supporter Certificate](https://sandboxie-plus.com/supporter-certificate/), see the table below.
+
+Starting with version 1.11.0 there is also a [Advanced Encryption Pack](https://xanasoft.com/product/sandboxie-plus-advanced-upgrade/) available which needs to be obtained in addition to a supporter certificate in order to use sandbox encryption.
```
```suggestion
Starting with version 1.11.0, an [Advanced Encryption Pack](https://xanasoft.com/product/sandboxie-plus-advanced-upgrade/) is also available, which must be obtained in addition to the supporter certificate in order to use sandbox encryption.
```

offhub committed:
Update 35044e8

moved files: (to Content)
sandboxielogon.md (renamed as SandboxieLogon.md)
feature-comparison.md (renamed as FeatureComparison.md)

deleted files:
changelog.md (unnecessary)
contribute.md (use CONTRIBUTING.md)
privacy-policy.md (website only)

Place holders:
PlusCert1.png
PlusCert2.png
PlusCert3b.png
PlusCert4.png
PlusCert5b.png
PlusCert6.png
sandbox-b-full.png
sandbox-c-full.png
sandbox-g-full.png
sandbox-k-full.png
sandbox-o-full.png
sandbox-r-full.png
sandbox-y-full.png

offhub committed:
fix errors

offhub committed:
Update Content/FeatureComparisonOld.md

Co-Authored-By: isaak654 <isaak654@users.noreply.github.com>

isaak654 commented on Content/FeatureComparison.md:
```diff
@@ -1,28 +1,34 @@
-| Security enhanced boxes          | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=FF0000&text=+) No      | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=green&text=+) Yes (no UI)  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=FF0000&text=+) No       | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=gr ...
-| Compatibility enhanced boxes    | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=FF0000&text=+) No      | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=green&text=+) Yes (no UI)  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=FF0000&text=+) No       | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=gre ...
-| Process Breakout                 | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=green&text=+) Yes (no UI)      | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=green&text=+) Yes (no UI)  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=green&text=+) Yes       | ![Green Color](https://placeholder.antonshell.me/img?width=15 ...
+Sandboxie Plus and Classic share the same core components, the main difference is that the Classic UI is no longer under development. Hence a UI for new core functionality is only available in the SandMan UI of the Sandboxie-Plus. Likewise various other new features are only implemented in the SandMan UI. Another difference is that Sandboxie Plus is provided under a [custom license](https://githu ...
```
```suggestion
Sandboxie Plus and Classic share the same core components, the main difference is that the Classic UI is no longer under development. Hence, a UI for new core functionality is only available in the SandMan UI of Sandboxie Plus. Likewise, various other new features are only implemented in the SandMan UI. Another difference is that Sandboxie Plus is provided under a [custom license](https://github.com/sandboxie-plus/Sandboxie/blob/master/Installer/license.txt), while Sandboxie Classic is provided under a [GPL-3.0+ license](https://github.com/sandboxie-plus/Sandboxie/blob/master/LICENSE).
```

offhub committed:
Update Content/FeatureComparison.md

Co-authored-by: isaak654 <isaak654@users.noreply.github.com>

isaak654 committed:
Update FeatureComparison.md

offhub committed:
Update FeatureComparison.md

isaak654 committed:
Update FeatureComparison.md

offhub committed:
Merge branch 'docsupdt004' of https://github.com/offhub/sandboxie-docs into docsupdt004

isaak654 reviewed (changes requested):
All uploaded .png files have a size of 120 bytes and cannot be displayed.

Part of them can be found in https://sandboxie-plus.com/applying-supporter-certificate/

offhub committed:
Update FeatureComparison.md

offhub committed:
missing images

isaak654 committed:
Update box-preset-comparison.md

isaak654 committed:
Update box-preset-comparison.md

isaak654 committed:
Update box-preset-comparison.md

isaak654 committed:
Update supporter-certificate.md

offhub committed:
add Sandboxie.md

isaak654 committed:
Update supporter-certificate.md

offhub committed:
add BoxEncryption.md

rename EncryptedBoxes.md to BoxEncryption.md

offhub committed:
fix pre

isaak654 committed:
Update applying-supporter-certificate.md

isaak654 committed:
Update FeatureComparisonOld.md

offhub committed:
update allpages.md

offhub committed:
fix pre

isaak654 committed:
Update FeatureComparisonOld.md

offhub committed:
Merge branch 'docsupdt004' of https://github.com/offhub/sandboxie-docs into docsupdt004

offhub committed:
Update AllPages.md

isaak654 committed:
Minor fixes

isaak654 committed:
Merge branch 'docsupdt004' of https://github.com/offhub/sandboxie-docs into pr/135

offhub committed:
Update privacy-mode.md

offhub committed:
Update security-mode.md

offhub committed:
Merge branch 'main' into docsupdt004

offhub committed:
allpages

offhub committed:
add ramdisksupport.md

offhub committed:
Update security-mode.md

isaak654 committed:
Update Content/FeatureComparison.md

isaak654 committed:
Update Content/FeatureComparison.md

isaak654 committed:
Update Content/FeatureComparison.md

isaak654 committed:
Minor fixes

offhub commented on Content/FeatureComparison.md:
```diff
@@ -1,28 +1,32 @@
+\*) A personal type certificate once expired does not unlock features in builds compiled after it expired, builds compiled before that time retain their exclusive unlocked features state.
+\*\*) Personal covers all devices you use yourself, so if you have a laptop and a desktop, one certificate covers both.
+\*\*\*) A family pack can be used for the entire family, it is not required for the family to live in the same household, so the certificate can be used for children who have moved out or your grandparents.
+\*\*\*\*) For as long as the service exists.
```
A new line should be added between the lines, otherwise it looks like a single line.

isaak654 committed:
Add new lines and trim trailing space

isaak654 committed:
Update FeatureComparison.md

Improve clarity

isaak654 committed:
Update FeatureComparison.md

Improve clarity

isaak654 committed:
Update Sandboxie.md

Capitalization fix

isaak654 committed:
Update UsePrivacyMode.md

Missing dot.

isaak654 committed:
Update UseRuleSpecificity.md

Missing dot.

isaak654 committed:
Update UseSecurityMode.md

Missing dot.

isaak654 commented on Content/AllPages.md:
```diff
@@ -38,8 +38,14 @@
+
+[Box Snapshots](../PlusContent/BoxSnapshots.md)
+
 ~~[Box Root Folder](BoxRootFolder.md)~~ (deprecated since Sandboxie v3.xx)
```
```suggestion
~~[Box Root Folder](BoxRootFolder.md)~~ (deprecated since Sandboxie v3.xx)

[Box Snapshots](../PlusContent/BoxSnapshots.md)
```

isaak654 commented on Content/AllPages.md:
```diff
@@ -70,6 +76,8 @@
 [Configure Menu](ConfigureMenu.md)
 
+[Compartment Mode](../PlusContent/compartment-mode.md)
+
```
This should be placed before `[Config Level](ConfigLevel.md)`.

isaak654 committed:
Update RamDiskSupport.md

offhub committed:
Update AllPages.md

isaak654 commented on Content/AllPages.md:
```diff
@@ -592,20 +620,32 @@
+
+[Use Rule Specificity](UseRuleSpecificity.md)
+
+[Use Security Mode](UseRuleSpecificity.md)
```
```suggestion
[Use Privacy Mode](UsePrivacyMode.md)

[Use Rule Specificity](UseRuleSpecificity.md)

[Use Security Mode](UseSecurityMode.md)

[User Accounts Settings](UserAccountsSettings.md)
```

offhub committed:
Update AllPages.md

offhub committed:
Update AllPages.md

isaak654 committed:
Update AllPages.md

Fix for mismatch link issue

isaak654 committed:
Multiple fixes

isaak654 commented on Content/UsePrivacyMode.md:
```diff
@@ -0,0 +1,17 @@
+# Privacy Mode
+
+_UsePrivacyMode_ is a box type preset available since v1.0.5 / 5.55.5.
```
```suggestion
_UsePrivacyMode_ is a box type preset available since v1.0.0 / 5.55.0.
```

Source: [CHANGELOG.md#100--5550---2021-11-17](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md#100--5550---2021-11-17)

isaak654 commented on .lycheeignore:
```diff
@@ -1,3 +1,5 @@
 https://www.ghacks.net/*
 https://web.archive.org/web/*
+https://www.paypal.com/en_AT/i/scr/pixel.gif
+https://www.paypal.com/cgi-bin/webscr
```
```suggestion
```

In my opinion, this should be committed if you think we're close to merge this PR (because here there is no longer a contribute.md file).

offhub replied:
I have nothing more to add to this pull request at the moment.

offhub committed:
Update Content/UsePrivacyMode.md

Co-authored-by: isaak654 <isaak654@users.noreply.github.com>

isaak654 committed:
Update BoxEncryption.md

Minor fixes

offhub committed:
Update .lycheeignore

Co-authored-by: isaak654 <isaak654@users.noreply.github.com>

isaak654 committed:
Last fixes and improvements
